backend/data/processors/conditions_preprocessing.py

```diff
@@ -518,5 +524,118 @@ def unsw_global_degrees(processed):
 
 
 
+def remove_extraneous_comm_data(processed):
+    """Adds handbook notes and removes phrasing
+    example: """
+    note = ""
+    # Academic Standing note
+    processed, number_of_subs = re.subn(
```

Comment:
this one and the next one can be collapsed into 1 line using regex:
```
r"(Students must be)? [io]n Good Academic Standing (and)?"
```
this can be done a lot for the following examples.